server/controllers/group_controller.js

```diff
@@ -63,25 +58,68 @@ Output: Returns json file of the group.
 // TODO: Clarify the point of this endpoint, should it get all the articles or
 // annotations, or be like a history/info about the group?
 export const getGroup = (groupId) => {
-  return Group.findOne({ _id: groupId });
+  return Group.findById(groupId);
+};
+
+/*
+Get a list of groups, filtered by some conditions.
+Input:
+  query: A mongodb query selector object
+Output: Resolves to a list of matching groups
+Example:
+  Groups.getGroupsFiltered({
+    members: user._id,
+    isPersonal: false,
+    articles: { $all: [article1ID, article2ID] },
+  });
+*/
+export const getGroupsFiltered = (query) => {
```

Comment:
Okay. I'm going to make a trello card that's just like "make queries consistent" across controllers so we can get that taken care of once we decide on how specific we want them to be.